compiler/fir/semantics/src/org/jetbrains/kotlin/fir/resolve/dfa/Flow.kt

```diff
@@ -5,8 +5,8 @@
 
 package org.jetbrains.kotlin.fir.resolve.dfa
 
-abstract class Flow {
-    abstract val knownVariables: Set<RealVariable>
-    abstract fun unwrapVariable(variable: RealVariable): RealVariable
-    abstract fun getTypeStatement(variable: RealVariable): TypeStatement?
+interface Flow {
```

Comment:
Ok, this was an accidental change anyway I think - at some point I wanted to rename PersistentFlow to Flow, but then manually reverted that and forgot that Flow was an abstract class rather than an interface.